foxpuppet/windows/browser/notifications/addons.py

```diff
@@ -33,9 +35,9 @@ def addon_name(self):
         """
         with self.selenium.context(self.selenium.CONTEXT_CHROME):
             if self.window.firefox_version >= 55:
-                return self.root.find_anonymous_element_by_attribute(
-                    'anonid', 'description').find_element(
-                        By.CSS_SELECTOR, '.addon-webext-name').text
+                return self.root.find_element(
+                    *self._web_ext_header_locator).find_element(
+                        *self._web_ext_name_locator).text
```

Comment:
Are you not able to reference this element directly using `#addon-webext-perm-header .addon-webext-name`? The only reason an intermediary node was used before was due to the parent node being anonymous.